src/absil/ilread.fs

```diff
@@ -3970,15 +3970,16 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-    if IL.runningOnWindows then
```

Comment:
memory mapped should work only on windows. They dont use the memorymapped bcl classes, instead use windows kernel api ( ref [source](https://github.com/fsharp/FSharp.Compiler.Service/blob/e3a3266bac7ec56f6de8b79a283c311ccbc99a56/src/absil/ilread.fs#L119) )
That a nice refactoring/improvement to do, move current implementation to [bcl MemoryMapped  .net classes](https://msdn.microsoft.com/en-us/library/dd997372(v=vs.110).aspx)